10.x core constructor property promotion

Blackbam opened the pull request:
Constructor property promotion for the core bundle (including tiny comment and code style improvements). Should not change any functionality, just saves development time and works better with IDEs.

Blackbam committed:
feat. constructor property promotion for commands

Blackbam committed:
feat. constructor prperty promotion and tiny code style fixes for the core bundle

dvesh3 commented on bundles/CoreBundle/EventListener/WorkflowManagementListener.php:
```diff
@@ -46,39 +46,12 @@ class WorkflowManagementListener implements EventSubscriberInterface
-    /**
-     * @var Registry
-     */
-    private $workflowRegistry;
```
$workflowRegistry missing in new contructor

Blackbam replied:
Yes. It was removed because it is a private property which is never called and therefore I assumed it is an artifact which is not needed anymore. Is there any benefit in adding it?

dvesh3 replied:
My bad, I didn't check for its usage. It is fine then we can remove it. Please remove the use on line 37

Blackbam replied:
@dvesh3 Removed (+ little type fix). Should fit now?

dvesh3 replied:
yes, thanks!

Blackbam committed:
Update WorkflowManagementListener.php

Remove unnecessary import of Registry class and add type decleration to $enabled.

brusch commented:
Thank you very much!
